web_src/fomantic/semantic.json

```diff
@@ -25,7 +25,6 @@
     "button",
     "checkbox",
     "container",
-    "dimmer",
```

Comment:
A quick search showed that we can also remove line `55` in `web_src/fomantic/theme.config.less` (at least that's how I understand this file)

Reply:
By the way, why do we still have less?

Reply:
Fomantic's source is Less and we compile that to CSS and vendor it. Theme config has no influence on compiled CSS, but I guess we can remove it there as well.

Reply:
So yes, it's definitely pointless to modify theme config. The file contains a bunch of modules we do not even build, but it really does not matter. We only consume `semantic.css` and `semantic.js` output files.